### ISSUE TITLE
UI - Make (+) in resource editor easier to click to create new entry

### ISSUE THREAD

**Issue by azaroth42** [archesproject/arches#6460]
**Is your feature request related to a problem? Please describe.**

To avoid having to click the blue "New" button, there is the (+) button in the resource editor. However the button only appears after hovering over the card in the left hand menu pane, which has two effects:

1. If you know what it does, then it increases the mouse precision required to first hover the menu item, then without leaving it, move the mouse over the pretty small (+) icon and then click it. This is bad for accessibility.
2. Because it doesn't appear until you hover, the user [in this case me] has probably already clicked the name. At which point clicking the "Add" is the same amount of RSI induction as clicking the (+)

**Describe the solution you'd like**

My non-expert UI idea: Make the + bigger, perhaps in a separate column, and always visible.
Any other solution would be great too.

**Comment by chiatt** [comment]
Not that this solves the problem, but I thought I should mention that there is also the `Add New` button in the upper right:
![image](https://user-images.githubusercontent.com/1877663/89595468-70620a00-d809-11ea-8d0f-26fa5ddf4230.png)

**Comment by azaroth42** [comment]
That's true. It doesn't show up for the first one, but there-after it's much easier to get to than the hovered (+)

What about if the entry card appeared for the first entry by default, rather than having an empty report view with the New button?  Then you could click the name, and go to add data. Then click Add New, and back to add more data.

**Comment by chiatt** [comment]
Yeah - I think that would be a nice improvement. The empty report doesn't provide a great deal of value over being able to start editing data right away.
